x-pack/plugins/apm/server/routes/errors/get_error_groups/get_error_group_sample.ts

```diff
@@ -41,6 +41,7 @@ export async function getErrorGroupSample({
       events: [ProcessorEvent.error as const],
     },
     body: {
+      track_total_hits: true,
```

Comment:
is there a default number of total hits in this case?

Reply:
No. The default is 10.000. Changing to `true` is unbounded and will count every hit. Potentially quite expensive but necessary in this case (unless we make UI changes which is out of scope for this PR).